components/composed/contributor/ContributorUpdatePersonForm/ContributorUpdatePersonForm.tsx

```diff
@@ -45,6 +71,7 @@ export default function ContributorUpdatePersonForm({ data }: Props) {
       mutation={mutation}
       getErrors={getErrors}
       toVariables={toVariables}
+      defaultValues={defaultValues}
```

Comment:
Hey @scryptmouse ! I was able to solve my type issue in ContributorLinksInput, but I'm getting an odd type error with the default values here. Error: https://vercel.com/cast-iron/wdp-admin-client/x9V7EpXW41rGEyPPTV7JKtbcT7TZ
```
Type error: Type '{ givenName?: string \| null \| undefined; familyName?: string \| null \| undefined; title?: string \| null \| undefined; email?: string \| null \| undefined; affiliation?: string \| null \| undefined; bio?: string \| ... 1 more ... \| undefined; links?: readonly ({ ...; } \| null)[] \| undefined; " $refType": "ContributorUpdateP...' is not assignable to type '{ title?: string \| null \| undefined; url?: string \| null \| undefined; email?: string \| null \| undefined; links?: { title?: string \| undefined; url?: string \| undefined; }[] \| null \| undefined; ... 5 more ...; clearImage?: boolean \| ... 1 more ... \| undefined; }'.
--
```
The error popped up after I moved the query vars out of the shared field fragment